config/environments/production.rb

```diff
@@ -30,15 +30,14 @@
   # Do not fall back to assets pipeline if a precompiled asset is missed.
   config.assets.compile = false
 
+  config.assets.compress = true
```

Comment:
I've moved those lines from the end of the file as here they're closer to the line above. All of those are assets related.